tcms/testplans/static/testplans/js/get.js

```diff
@@ -376,6 +377,26 @@ function attachEvents (testPlanId, permissions) {
         })
     }
 
+    if (permissions['perm-view-user']) {
+        $('#default-tester-input').typeahead({
+            minLength: 3,
+            highlight: true
+        }, {
+            name: 'default-tester-autocomplete',
+            // will display up to X results even if more were returned
+            limit: 100,
+            async: true,
+            display: function (element) {
+                return element.username
+            },
+            source: function (query, processSync, processAsync) {
+                jsonRPC('User.filter', { username__icontains: query }, function (data) {
+                    return processAsync(data)
+                })
+            }
+        })
+    }
```

Comment:
A few issues with how this function/modal behaves:

Issue #1

1) Use it once to assign a default tester. I chose "alice" then
2) Mark some test cases and trigger the default tester modal again -> the text "alice" is still present inside the input field, it should be empty field instead


Issue #2

Every time the modal opens the typeahead handler gets initialized again. Using the modal 5-6 times without reloading the page leads to a chain of `User.filter` RPC calls instead of having only 1 call. This visibly slows down the page. This is similar to #3281 .